### DISCUSSION TITLE
Github education program applying process is awful

### DISCUSSION THREAD

**Discussion by pilotocheg** [community/community#141667] [GitHub Education]
### Hi everyone,

I was rejected 3 times. There's no option to attach any electronic documents/screenshots/images, only to take pictures with device's camera. And even that is suck, because I can't use rear camera on my iPhone. When I select "rear camera" option from a dropdown there's no camera view. So I need to take pictures only with front camera somehow. So only way to attach electronical documents is to take pictures of them opened on computer with my iphone's front camera.

**Comment by BrunoL0M** [comment]
desde mi perspectiva fue la verdad muy facil, el problema que tuve una y otra vez el cual no me aceptaba mi evidencia, era por que no cumplia con los requisitos el cual tiene que tener una cuenta de github que es basicamente , 2AF, poner la direccion de tu domicilio, y tambien creo que si no me equivoco tambien era poner tu correo universitario, estudiantil, etc. en alguna seccion de la configuracion de perfil, una vez eso me tomo mi evidencia de la universidad y pude acceder facilmente, y no era una prueba tan solida que digamos

**Comment by frangoni** [comment]
I agree it is a disaster. It is required to add a translation of the original document but you can take only one picture and adding two documents on a single page makes it impossible to read.
